onnx_chainer/functions/array.py

```diff
@@ -119,8 +119,18 @@ def convert_GetItem(func, opset_version, input_names,
                 'ONNX-Chainer does not accept the type'.format(type(idx)))
 
     gb = onnx_helper.GraphBuilder()
-    output = gb.op('Slice', input_names,
-                   axes=axes, starts=starts, ends=ends)
+    if opset_version == 10:
+        starts_param = chainer.Parameter(
+            np.asarray(list(starts), dtype=np.int64))
+        ends_param = chainer.Parameter(np.asarray(list(ends), dtype=np.int64))
+        axes_param = chainer.Parameter(np.asarray(list(axes), dtype=np.int64))
+        for p in [starts_param, ends_param, axes_param]:
```

Comment:
how about `for param in [starts, ends, axes]: ...` and aggregate `chainer.Parameter` conversion?